### ISSUE TITLE
50728073: Incorrect Voiceover text for pickers when language is French

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#21256]
#### Description

Area:
Accessibility

Summary:
When iOS device language is set to French and Voiceover is switched on, when a user is accessing a picker, the Voiceover says (in French) to "swipe left or right to adjust" whereas it should be "swipe up or down to adjust".

Steps to Reproduce:
1. iOS Settings > General > Accessibility > VoiceOver > switch ON
2. General > Language & Region > iPhone Language > select "Francais / French" and then select "Change to French".
3. Go to Home screen and open "Horloge" app ("Clock")
4. Select "Minuteur" tab ("Timer")
5. Swipe right to highlight either the hour, minute or second picker.
6. Listen to the Voiceover for the highlighted picker.

Expected Results:
The instructions for using the picker should say (in French) "Swipe up or down to adjust"

Actual Results:
The instructions for using the picker says (in French) "Swipe left or right to adjust"

Version/Build:
iOS 12.2

Configuration:
Language = French
Voiceover ON
iPhone 6S / iOS 12.2

-
Product Version: 12.2
Created: 2019-05-13T17:26:57.622254
Originated: 2019-05-13T00:00:00
Open Radar Link: http://www.openradar.me/50728073
